roles/confluent.common/tasks/main.yml

```diff
@@ -46,7 +46,7 @@
     group: "{{ omit if archive_group == '' else archive_group }}"
     owner: "{{ omit if archive_owner == '' else archive_owner }}"
     mode: 0755
-    creates: "{{archive_destination_path}}/confluent-{{confluent_package_version}}"
+    creates: "{{archive_destination_path|regex_replace('\\/$', '')}}/confluent-{{confluent_package_version}}"
```

Comment:
can you test out `path_join` filter instead? https://docs.ansible.com/ansible/latest/user_guide/playbooks_filters.html

Reply:
i dont remember the reason i didnt use it in the past, might have not been included w an older ansible version